symfony/src/Repository/MeasurementRepository.php

```diff
@@ -100,4 +100,24 @@ public function getUnloggedMissingMeasurements(): array
             'type' => MissingNotificationLog::MEASUREMENT_TYPE
         ]);
     }
+
+    public function getMeasurement($measurementId, $participantId)
+    {
+        $parentIds = $this->createQueryBuilder('m')
```

Comment:
When there are no PMs with parent ids, this returns an empty array, and the not in clause below results in the measurement not being found. I know this is the way it currently is pre-migration, but even if we fixed this to handle the empty result set, it's not ideal that we're having to load every modified PM id into an array to exclude them.

Can you remind me what the reason is for not returning a PM that has been superceded by another? Do we not allow the user to view these?

Reply:
We did this to mirror what we are doing for orders modifications. Since we are creating a new PM instead of modifying the existing one we decided to hide the parent PM.

Reply:
I went ahead and fixed the empty parent ids issue for now. We can revisit and see what would be a better solution for this.